ui/plugins/job_details_pane.jsx

```diff
@@ -0,0 +1,264 @@
+
+    return <ul className="list-unstyled content-spacer">
+        <li>
+            <span title={classificationName.name}><i className={`fa ${iconClass}`}></i>
+            {classificationName.name}</span>
+            {props.bugs.length > 0 &&
+            <a target="_blank" href={props.getBugUrl(props.bugs[0].bug_id)}
+            title={`View bug ${props.bugs[0].bug_id}`}><em> {props.bugs[0].bug_id}</em></a>}
+        </li>
+        {props.classifications[0].text.length > 0 &&
+        <li><em dangerouslySetInnerHTML={repoURLHTML}></em></li>}
+        <li className="revision-comment">
+            {props.dateFilter(props.classifications[0].created, 'EEE MMM d, H:mm:ss')}</li>
+            <li className="revision-comment">
+            {props.classifications[0].who}
+        </li>
+    </ul>;
+}
+
+
+const JobStatusPane = props => (
+    <ul className="list-unstyled">
+        <li id="result-status-pane" className={`small ${props.resultStatusShading}`}>
+            <div>
+                <label>Result:</label>
+                <span> {props.job.result}</span>
+            </div>
+            <div>
+                <label>State:</label>
+                <span> {props.job.state}</span>
+            </div>
+            {props.job.state === 'running' && <div>
+                {props.eta > 0 && <span>Time remaining: ~{props.eta} minutes</span>}
+                {props.eta < 0 && <span>{Math.abs(props.eta)} mins overdue, typically takes ~{props.averageDuration} mins</span>}
+                {props.eta === 0 && <span>Due any minute now, typically takes ~{props.averageDuration} mins</span>}
+            </div>}
+            {props.job.state === 'pending' && <div>
+                <span>Typically takes ~{props.averageDuration} mins once started</span>
+            </div>}
+        </li>
+    </ul>
+);
+
+
+const JobDetailsListItem = props => (
+    <li className="small">
+        <label>{props.label}</label>
+        {props.labelHref &&
+        <a title={props.labelTitle}
+            href={props.labelHref}
+            onClick={props.labelOnclick}
+            target={props.labelTarget}>
+            {props.labelText} <span className="fa fa-pencil-square-o icon-superscript"></span>: </a>}
+        {!props.href ? <span className="ml-1">{props.text}</span> :
+        <a title={props.title}
+            className="ml-1"
+            href={props.href}
+            onClick={props.onclick}
+            target={props.target}>
+            {props.text}</a>}
+        {props.iconClass && <span className={`ml-1${props.iconClass}`}></span>}
+    </li>
+);
+
+
+class JobDetailsList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.filterTextEvent = this.filterTextEvent.bind(this);
+    }
+
+    filterTextEvent(event, input) {
+        event.preventDefault();
+        this.props.filterByJobSearchStr(input);
+    };
+
+    render() {
+        let jobObj = this.props.job;
```

Comment:
I think a few of these can be ``const`` if they're not reassigned.  Like ``jobBuildSystem``.  On the other hand, maybe you don't need these, since the reference of just ``jobObj.build_system_type`` isn't that long.  

I'd actually be inclined to rename ``jobObj`` to just ``const job``.